web/app.js

```diff
@@ -486,6 +486,11 @@ const PDFViewerApplication = {
           appConfig.toolbar?.editorStampButton?.classList.add("hidden");
         }
 
+        const editorHighlightButton = appConfig.toolbar?.editorHighlightButton;
+        if (editorHighlightButton && AppOptions.get("enableHighlightEditor")) {
+          editorHighlightButton.hidden = false;
```

Comment:
To reduce the risk of this button "flashing by" when the viewer is loading, should we perhaps hide it by default (in the HTML) and instead *show* it here?

Reply:
I think it's already the case:
https://github.com/mozilla/pdf.js/pull/17317/files#diff-4b300cea2df7f3f32d97ec5bd2e71eee41932f0fb52f31516f91dbff0c683409R350